packages/neuron-wallet/src/main.ts

```diff
@@ -76,10 +76,14 @@ function createWindow() {
   i18n.changeLanguage(app.getLocale())
   Menu.setApplicationMenu(mainmenu())
 
-  mainWindow.loadURL(env.mainURL)
+  const url = WalletsController.activeOne().status ? env.mainURL : `${env.mainURL}/wallets/wizard`
```

Comment:
Feels we should have a fixed root URL in `neuron-wallet` and let the UI to determine which actual route/path to display on start up.

Reply:
UI will handle the case of an empty wallet too, but it's asynchronous that makes the app launch unnatural(view flash or something)

Reply:
Then what if based on data or logic the start screen could be more than just current `home` or `wizard`?

The nature of the web implementation would force us to have a better approach. Some considerations:

* Make it synchronous for certain calls. For example, checking if a wallet exist perhaps should support synchronous check.
* On the UI layer we show a start up blank screen with loading indicator, and switch to home (active wallet) or wizard or any other views we want to add in the future.

Reply:
Please leave that to a future plan or discussion.

Reply:
I'd like not to use a synchronous call cause it makes render blocked(I've tried in sendCapacity, the synchronous call should be put to the end of events queue manually). But a launch screen is a good idea, I'll add it soon.